src/browser/modules/Stream/SchemaFrame.jsx

```diff
@@ -79,49 +129,26 @@ export class SchemaFrame extends Component {
     }
   }
 
-  formatIndexAndConstraints (indexes, constraints) {
-    let indexString
-    let constraintsString
-
-    if (indexes.length === 0) {
-      indexString = 'No indexes'
-    } else {
-      indexString = 'Indexes'
-      indexString += indexes.reduce((acc, index) => {
-        acc += `\n  ${replace(
-          index.description,
-          'INDEX',
-          ''
-        )} ${index.state.toUpperCase()} ${
-          index.type === 'node_unique_property'
-            ? ' (for uniqueness constraint)'
-            : ''
-        }`
-        return acc
-      }, '')
-    }
-
-    if (constraints.length === 0) {
-      constraintsString = 'No constraints'
-    } else {
-      constraintsString = 'Constraints'
-      constraintsString += constraints.reduce((acc, constraint) => {
-        acc += `\n  ${constraint.description.replace('CONSTRAINT', '')}`
-        return acc
-      }, '')
-    }
+  render () {
+    const indexes = formatIndexes(this.state.indexes)
+    const constraints = formatConstraints(this.state.constraints)
 
-    return `${indexString}\n\n${constraintsString}\n`
-  }
+    const frame = (
+      <Slide>
+        <SchemaTable name='Indexes' content={indexes} />
+        <SchemaTable name='Constraints' content={constraints} />
+        <br />
+        <p className='lead'>Execute the following query for more information</p>
+        <figure>
+          <pre className='code runnable'>CALL db.schema.visualization</pre>
```

Comment:
neo4j-browser needs (at the time of writing this) to support Neo4j 3.4+, and `db.schema.visualization` does not exist in Neo4j 3.4. There it's named `db.schema()`, so you need to do a version check here.